python/mxnet/initializer.py

```diff
@@ -79,17 +110,27 @@ def __call__(self, desc, arr):
         if init:
             # when calling Variable initializer
             create(init)._init_weight(desc, arr)
+            if self._verbose:
+                logging.info('Initialized %s by %s: %s', desc, init, self._print_helper(arr))
         else:
             # register nnvm::FSetInputVariableAttrs in the backend for new patterns
             # don't add new cases here.
             if desc.endswith('weight'):
                 self._init_weight(desc, arr)
+                if self._verbose:
```

Comment:
Can we use python descriptor to implement this? It seems there are bunches of repeated codes.